examples/tensorflow/classification/main.py

```diff
@@ -18,6 +18,7 @@
 import tensorflow as tf
 import tensorflow_addons as tfa
 
+from examples.tensorflow.common.utils import close_strategy_threadpool
```

Comment:
Could you please move this import to the `examples.` group of imports below?

Reply:
Sure, thanks. Pycharm bug. Did not check import placement.